.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+        default=None,
+        const=None,
+        help="""An optional space separated list of \
+directories to exclude from the analysis.""",
+    )
+
+    # Return parser
+    result = parser.parse_args()
+    return result
+
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)